### ISSUE TITLE
MO 2.0.6

### ISSUE THREAD

**Issue by Mabfefe** [TanninOne/modorganizer#754]
Just updated to new beta release. When launching 4FSE through MO game will load up until the menu screen before freezing. FO4 runs just fine from steam and when I run 4FSE outside of MO.

**Comment by Mabfefe** [comment]
So it seems that F4SE, LOOT, and FO4Edit are not able to be executed from within MO 2.0.6b. I reverted back to MO 2.0.5b and only FO4Edit seems to be having problems as it creates some very strange files that need to be deleted before MO 2.0.5b will launch FO4.

**Comment by azchar111** [comment]
I had the same issue with F4SE but a restart of MO 2.06b allowed F4SE to run normally.  Had no issue with LOOT 0.9.0 (build 44a0d85).  Attempting to start F4SE a few times (3-4) times also gets it running for me.

**Comment by Mabfefe** [comment]
Yeah. That was the same problem I was running into. At first I thought it was my anti virus software that was attempting to block some of the programs from initiating, but it occurred even when I had the software turned off.

**Comment by Kitsudragon** [comment]
Also had the same problem. I disabled all the mods entirely. Just FO4 with a blank MO list. I can start FO4 with F4SE no problem. But the moment I use MO, it crashes at the start screen. I don't even get to the Bethesda animated graphic.

(No mods loaded. Just running a blank MO.)
